kuma/landing/templates/landing/homepage.html

```diff
@@ -192,4 +203,15 @@ <h2 class="title">{{ _('Get Involved') }}</h2>
         jQuery('#home-search-form').searchSuggestions();
     </script>
   {% endif %}
+
+  {# This can be removed once the homepage heading A/B test is completed. #}
+  <script>
+      {% if not waffle.flag('hide_homepage_heading') %}
+          // Bucket visitor into the baseline Optimizely variation for reporting
+          mdn.optimizely.push(['bucketVisitor', 2702600192, 2707400176]);
+      {% else %}
+          // Bucket visitor into the tested Optimizely variation for reporting
+          mdn.optimizely.push(['bucketVisitor', 2702600192, 2709140171]);
+      {% endif %}
+  </script>
```

Comment:
[Using the Optimizely API to avoid flashes of unstyled content in A/B tests](http://blog.openjck.com/using-the-optimizely-api-to-avoid-flashes-of-unstyled-content-in-ab-tests/)